src/components/TipJar/TipJar.js

```diff
@@ -1,10 +1,22 @@
-import { description } from 'commander';
 import React from 'react';
 import TipCard from '../TipCard/TipCard'
 import './TipJar.css';
 
 const TipJar = ({ tips, handleDelete }) => {
-  const allTips = tips.map(tip => {
+
+  const miliSecTips = tips.map(tip => { 
+    return { ...tip, date: Date.parse(tip.date) }
+  }).sort((a, b) => b.date - a.date)
+
+  const formatDate = (date) => {
+    return new Date(date).toLocaleString('en-US', { year: 'numeric', month: 'numeric', day: 'numeric' })
```

Comment:
Hmm seems like we are converting back and forth between formatting...maybe our data should just be in milliseconds formatting